www/js/app.js

```diff
@@ -89,29 +89,30 @@ function handleOpenURL(url) {
     
     if (kvList.route == 'join') {
       var INTRO_DONE_KEY = 'intro_done';
+      var HABITICA_REGISTERED_KEY = 'habitica_registered';
+      var REFERRAL_NAVIGATION_KEY = 'referral_navigation';
+      var REFERED_KEY = 'refered';
+      var REFERED_GROUP_ID = 'refered_group_id';
+      var REFERED_USER_ID = 'refered_user_id'
+      window.localStorage.setItem(REFERED_KEY, true);
```

Comment:
why not use the `angularLocalStorage` service? that's what `startprefs.js` does...